api/models.py

```diff
@@ -284,3 +284,6 @@ class Infraction(ModelReprMixin, models.Model):
         default=False,
         help_text="Whether the infraction is a shadow infraction."
     )
+
+    def __str__(self):
+        return self.type
```

Comment:
I think this should be a bit more descriptive, I'm not 100% sure about whether this is shown in the admin or whether it's `__repr__` but the type on its own doesn't really tell much. Maybe something like
```py
return f"#{self.id}: {self.type} on {self.user or 'unknown user'}"
```
and maybe some more magic with active, or expiry?